### DISCUSSION TITLE
How to return cutted tiles

### DISCUSSION THREAD

**Discussion by DiFFoZ** [SmartlyDressedGames/Unturned-3.x-Community#3176] [Q&A] [answered]
After using `Cut` instrument and after that save and exit from editor it's now not possible to use `CTRL + Z` to return not cutted terrain back.

So, maybe there's has an instrument to return back cutted terrain? Because it is annoying to delete everything and make again cutting terrain.

**Comment by SDGNelson** [comment, marked as answer]
If you hold `Shift` you can restore cut portions of the terrain. Maybe I should add a hint to the UI for this.
